buildspec-pr.yaml

```diff
@@ -12,6 +12,10 @@ phases:
 
       # Install yarn if it wasn't already present in the image
       - yarn --version || npm -g install yarn
+
+      # Packing the mono-libraries (monocdk & aws-cdk-lib) can cause
+      # memory errors. Increasing this value allows our build to more consistently succeed
+      - yum install -y procps-ng && /sbin/sysctl -w vm.max_map_count=2251954
```

Comment:
Maybe already make this apt-safe for the future? Like only do the `yum` dance if `command -v sysctl` is non-success?

Reply:
Good call :+1: